test/functional/feature_asmap.py

```diff
@@ -72,6 +80,15 @@ def test_default_asmap(self):
                 self.start_node(0, [arg])
         os.remove(self.default_asmap)
 
+    def test_asmap_interaction_with_addrman_containing_entries(self):
+        self.log.info('Test bitcoind -asmap with an addrman containing new and tried entries')
+        self.fill_addrman(node_id=0)
```

Comment:
Note that the addresses will get added to the new/tried table according to the asmap that was loaded in the previous subtest, which is currently `test_default_asmap()`. If the ordering of the tests changes, then the asmap used for the previous test may be different from this, and the addresses wouldn't necessarily be rebucketed in the same locations.

Reply:
Good point. Do you think this would be worthwhile?
```diff
     def test_asmap_interaction_with_addrman_containing_entries(self):
         self.log.info("Test bitcoind -asmap with an addrman containing new and tried entries")
+        self.stop_node(0)
+        shutil.copyfile(self.asmap_raw, self.default_asmap)
+        self.start_node(0, ["-asmap"])
         self.fill_addrman(node_id=0)
         self.stop_node(0)
```

Reply:
updated

Reply:
:+1: that would make the subtests independent